compiler/rustc_hir_analysis/src/check/wfcheck.rs

```diff
@@ -105,7 +105,7 @@ where
     if !tcx.features().trivial_bounds {
         wfcx.check_false_global_bounds()
     }
-    f(&mut wfcx);
+    f(&mut wfcx)?;
```

Comment:
Note that this causes any method receiver errors to prevent the trait solver from emitting any errors below